src/components/pagination/__test__/pagination.test.js

```diff
@@ -31,6 +31,10 @@ describe('Pagination component', () => {
     const component = renderer.create(<Pagination showPrevNext={false} delta={3} total={5} current={2} />);
     expect(component.toJSON()).toMatchSnapshot();
   });
+  it('Should not display First/Last page buttons', () => {
+    const component = renderer.create(<Pagination showFirstLast={false} delta={3} total={5} current={2} />);
```

Comment:
This test do not seem to be testing the feature, because the with delta=3 the pagination will display from 1 to 5 page with and without the new prop. 
 
also could you also add the `showFirstLast={true} ` test? there are a  few cases to check, (display the first but not the last, display the first and the last, and so on.